### DISCUSSION TITLE
LDAP required?

### DISCUSSION THREAD

**Discussion by manwithpowers** [snipe/snipe-it#9084] [Q&A] [answered]
In the [Requirements](https://snipe-it.readme.io/docs/requirements) for Snipe-IT, LDAP is listed as required "only if using LDAP", however I am unable to install Snipe-IT without it. I will not be using LDAP authentication and I would like to install Snipe-IT on my shared hosting account (Dreamhost), where I'm unable to install the LDAP extension.
Is there a way to install Snipe-IT without LDAP?

Thanks in advance.

**Comment by uberbrady** [comment, marked as answer]
Not at this time, no :(

But whenever we revamp the LDAP setup - which we are at least talking about doing - I will make sure that it only requires LDAP if you're actually using LDAP. That's a small ask, and it certainly makes sense to me.

As a workaround, you can use the latest 4.x and it will work perfectly fine without the LDAP extension being installed.

**Reply by manwithpowers** [reply]
Thanks very much.
I'll give the latest 4.x version a shot in the meantime.
